modules/bbp-morphology/src/main/resources/schemas/bbp/morphology/acquisitionannotation/v0.1.0.json

```diff
@@ -0,0 +1,59 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/prov/rec-prov-o-20130430/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "bbpentsh": "{{base}}/schemas/bbp/core/activity/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/experiment/acquisitionannotation/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "owl:Ontology",
+  "imports": [
+    "{{base}}/schemas/bbp/core/activity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id" : "this:AcquisitionAnnotationShape",
+      "@type" : "sh:NodeShape",
+      "description" : "Acquisition and annotation shape definition. This shape should have an IRI as identifier.",
```

Comment:
Not before submission to nexus-kg. The "nodeKind" : "sh:BlankNodeOrIRI" constraint assert that a valid AcquisitionAnnotation instance can have an identifier or just be a blank node.

Reply:
Ok removed  This shape should have an IRI as identifier